dev/tidb-in-kubernetes/deploy/prerequisites.md

```diff
@@ -71,6 +71,48 @@ If the above command shows that the swap column is all `0`, then swap is disable
 
 In addition, to permanently disable swaps, remove all the swap-related entries in `/etc/fstab`.
 
+After all the above configurations are made, check whether you have configured [SMP IRQ Affinity](https://cs.uwaterloo.ca/~brecht/servers/apic/SMP-affinity.txt) on the machine. This configuration is to assign the interrupt of each device to different CPUs to prevent all interrupts from being sent to the same CPU, avoiding the potential performance bottleneck. For the TiDB cluster, the rate at which the network card processes packages has a great impact on the throughput of the cluster. Use the following methods to assign the interrupt of a network card to a specific CPU, thus taking advantage of the multiple cores to increase cluster throughput.
+
+First execute the following command to check the interrupt of a network card:
+
+{{< copyable "shell-regular" >}}
+
+```shell
+cat /proc/interrupts|grep <iface-name>|awk '{print $1,$NF}'
+```
+
+In the output result of the above command, the first column indicates the interrupt and the second column indicates the device name. If it is a multi-queue network card, the above command outputs information in multiple rows and each queue corresponds to an interrupt. Use the following command to check this interrupt is assigned to which CPU:
+
+{{< copyable "shell-regular" >}}
+
+```shell
+cat /proc/irq/<ir_num>/smp_affinity
+```
+
+The above command outputs the hexadecimal value corresponding to the CPU serial number, and the output result is not so intuitive. For the detailed calculation method, refer to [SMP IRQ Affinity](https://cs.uwaterloo.ca/~brecht/servers/apic/SMP-affinity.txt).
+
+{{< copyable "shell-regular" >}}
+
+```shell
+cat /proc/irq/<ir_num>/smp_affinity_list
+```
+
+The above command outputs the decimal value corresponding to the CPU serial number, and the output result is more intuitive.
+
+If all interrupts of a multi-queue network card are assigned to different CPUs, the SMP IRQ Affinity is correctly configured on the machine. If all interrupts are sent to the same CPU, make adjustment according to the following two methods. These methods are for the scenario of multi-queue network card and multiple cores.
```

Comment:
I doubt whether the "These methods are for the scenario of multi-queue network card and multiple cores." should be moved to the beginning. Please check.